actions/api/community_events.py

```diff
@@ -54,53 +59,54 @@ def as_kwargs(self) -> Dict[Text, Text]:
             "event_date": self.formatted_date,
         }
 
+    @staticmethod
+    def parse_community_date(date_string: Text) -> datetime.datetime:
 
-def parse_community_date(date_string: Text) -> datetime:
-
-    dates = date_string.split("-")
-
-    try:
-        return datetime.strptime(dates[-1].strip(), DATE_FORMAT)
-    except Exception as e:
-        logger.warning(e)
-        return datetime.min
-
+        dates = date_string.split("-")
 
-def get_community_events() -> List[CommunityEvent]:
-    """Returns list of community events sorted ascending by their date."""
-    from bs4 import BeautifulSoup
-    import requests as r
-    import datetime
-
-    response = r.get("https://rasa.com/community/join/")
+        try:
+            return datetime.datetime.strptime(dates[-1].strip(), DATE_FORMAT)
+        except Exception as e:
+            logger.warning(e)
+            return (
+                datetime.datetime.max
+            )  # if date can't be parsed assume event is future
 
-    if response.status_code == 200:
-        community_page = response.content
+    @staticmethod
+    def get_community_page() -> requests.Response:
+        return requests.get("https://rasa.com/community/join/")
```

Comment:
makes sense to pull this out, can we just make this a constant at the top of the module so we know where to find it if it changes?